### PR TITLE
Charlesmchen/attachment type assets

### PR DESCRIPTION
**Pull request by charlesmchen-signal**
PTAL @michaelkirk 

img_0050

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/Cells/OWSGenericAttachmentView.m [hunk 1]
```diff
@@ -118,62 +118,31 @@ - (void)createContents
...
-        [MIMETypeUtil fileExtensionForMIMEType:self.attachmentStream.contentType];
-    }
-    if (fileExtension.length < 1) {
-        fileExtension = NSLocalizedString(@"GENERIC_ATTACHMENT_DEFAULT_TYPE",
-            @"A default label for attachment whose file extension cannot be determined.");
-    }
-
-    UILabel *fileTypeLabel = [UILabel new];
-    fileTypeLabel.text = fileExtension.uppercaseString;
-    fileTypeLabel.textColor = iconColor;
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSGenericAttachmentView.m]: So we no longer include the file extension in the file icon? (e.g. PDF, etc)

File: Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m [hunk 2]
```diff
@@ -315,6 +327,28 @@ - (nullable NSString *)fileTypeForSnippet
     return nil;
 }
 
+- (BOOL)isAudioAttachment
```
Comment (michaelkirk-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m]: So we use the mic for audio files *and* voice notes?

File: Signal/src/ViewControllers/ConversationView/Cells/OWSGenericAttachmentView.m (hunk 1 above)
Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSGenericAttachmentView.m]: Yes. My understanding is that Myles is already working on a new design for this but its not ready yet.

File: Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m (hunk 2 above)
Comment (charlesmchen-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m]: Yes. We can't distinguish them here, right?

**Pull request merged**